### PR TITLE
Update Instance in Guide to mention Object.freeze

### PR DESCRIPTION
**Pull request by sdras**
This PR clarifies that Object.freeze keeps Vue's reactivity system from updating properties, as per #1319 and #978. Open to suggestions for placement in guide. Also, as per conversation in chat, adding in `yarn.lock`

### CODE REVIEW

**Comment by sdras** [comment]
Apologies for the formatting changes, prettier has some opinions on those js code block spaces :)

File: src/v2/guide/instance.md [hunk 1]
```diff
@@ -103,12 +130,12 @@ Each Vue instance goes through a series of initialization steps when it's create
...
 For example, the [`created`](../api/#created) hook can be used to run code after an instance is created:
 
-``` js
+```js
 new Vue({
   data: {
     a: 1
   },
-  created: function () {
+  created: function() {
```
Comment (yyx990803) [inline src/v2/guide/instance.md]: it'd be great to revert the stylistic changes just for consistency

**Comment by yyx990803** [comment]
LGTM once we remove the two small stylistic changes 👍

**Comment by sdras** [comment]
Wow, you're speedy Chris, I was just fixing that up. Thanks!

**Comment by chrisvfritz** [comment]
I'm speedy in spurts. 😅 Today is a heavy open source day for me.

**Pull request merged**